@steveharter @LakshanF - does this failure make sense when picking up Preview7 SDK?
```
  [Trimming Tests] Running test: /__w/1/s/src/libraries/System.ObjectModel/tests/TrimmingTests/TypeConverterAttributeStringArgCtorTest.cs ...
  MONO_WASM: Runtime instantiation of this attribute is not allowed.
     at System.ComponentModel.DefaultValueAttribute.get_Value()
     at TypeConverterAttributeTest.Program.Main(String[] args)
     at System.Reflection.MethodBaseInvoker.InterpretedInvoke_Method(Object obj, IntPtr* args)
     at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)
EXEC : error : Runtime instantiation of this attribute is not allowed. [/__w/1/s/src/libraries/System.ObjectModel/tests/TrimmingTests/System.ObjectModel.TrimmingTests.proj]
##[error]EXEC(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Runtime instantiation of this attribute is not allowed.
```
I see you've been doing work here with https://github.com/dotnet/runtime/pull/105766 so I don't want to jump to assumptions on the fix.
https://github.com/dotnet/runtime/blob/ae49f9e502af2c540a21746abda8334f68fec4d9/src/libraries/System.ObjectModel/tests/TrimmingTests/TypeConverterAttributeStringArgCtorTest.cs#L20

https://dev.azure.com/dnceng-public/public/_build/results?buildId=774284&view=logs&j=7b499086-9930-5591-220e-2ec7c40eb990&t=016d048d-94ae-5266-5657-3d7f9fdefeae&l=551

--- a/src/libraries/System.ObjectModel/tests/TrimmingTests/System.ObjectModel.TrimmingTests.proj
+++ b/src/libraries/System.ObjectModel/tests/TrimmingTests/System.ObjectModel.TrimmingTests.proj
@@ -1,9 +1,15 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
-  <PropertyGroup>
-    <_DefaultValueAttributeSupport>true</_DefaultValueAttributeSupport>
-  </PropertyGroup>
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="TypeConverterAttributeStringArgCtorTest.cs">
+      <EnabledProperties>_DefaultValueAttributeSupport</EnabledProperties>
+    </TestConsoleAppSourceFiles>
+    <TestConsoleAppSourceFiles Include="TypeConverterAttributeTypeArgCtorTest.cs">
+      <EnabledProperties>_DefaultValueAttributeSupport</EnabledProperties>
+    </TestConsoleAppSourceFiles>
+    <TestConsoleAppSourceFiles Include="TypeDescriptionProviderAttributeCtorTest.cs" />
+  </ItemGroup>
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets))" />
 </Project>

--- a/src/libraries/System.ObjectModel/tests/TrimmingTests/System.ObjectModel.TrimmingTests.proj
+++ b/src/libraries/System.ObjectModel/tests/TrimmingTests/System.ObjectModel.TrimmingTests.proj
@@ -1,5 +1,9 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
+  <PropertyGroup>
+    <_DefaultValueAttributeSupport>true</_DefaultValueAttributeSupport>
+  </PropertyGroup>
+
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets))" />
 </Project>

--- a/src/libraries/System.Runtime/tests/System.Runtime.Tests/TrimmingTests/System.Runtime.TrimmingTests.proj
+++ b/src/libraries/System.Runtime/tests/System.Runtime.Tests/TrimmingTests/System.Runtime.TrimmingTests.proj
@@ -24,7 +24,9 @@
     <TestConsoleAppSourceFiles Include="DebuggerVisualizerAttributeTests.cs" >
       <EnabledProperties>DebuggerSupport</EnabledProperties>
     </TestConsoleAppSourceFiles>
-    <TestConsoleAppSourceFiles Include="DefaultValueAttributeCtorTest.cs" />
+    <TestConsoleAppSourceFiles Include="DefaultValueAttributeCtorTest.cs">
+      <EnabledProperties>_DefaultValueAttributeSupport</EnabledProperties>
+    </TestConsoleAppSourceFiles>
     <TestConsoleAppSourceFiles Include="GenericArraySortHelperTest.cs" />
     <TestConsoleAppSourceFiles Include="InheritedAttributeTests.cs" />
     <TestConsoleAppSourceFiles Include="InterfacesOnArrays.cs" />

